### PR TITLE
feat: add link to storybook, add showIframe and showButton booleans

### LINKED ISSUE #161: Add Link to storybook
Add showIframe and showButton boolean

### PR DESCRIPTION
**Pull request by pyphilia**
close #161

### CODE REVIEW

File: src/items/LinkItem.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { EmbeddedLinkItemExtra, ImmutableItemClass } from '../types';
+import { TABLE_CATEGORIES } from '../utils/storybook';
+import LinkItem from './LinkItem';
+import { Item, ItemType } from '@graasp/sdk';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { RecordOf } from 'immutable';
+import React from 'react';
```
Comment (spaenleh) [inline src/items/LinkItem.stories.tsx:7]: This repo is not using the sorted imports plugin ? We could add it (maybe in a next PR)

File: src/items/LinkItem.tsx [hunk 2]
```diff
@@ -54,12 +71,15 @@ const LinkItem: FC<LinkItemProps> = ({
...
   showCaption = true,
+  showIframe = true,
+  showButton = false,
   loadingMessage = 'Link is Loading...',
   openLinkMessage = 'Click here to open the link manually',
   height: defaultHeight,
   errorMessage = 'The link is malformed.',
   isResizable = false,
 }) => {
+  console.log(showIframe);
```
Comment (spaenleh) [inline src/items/LinkItem.tsx:82]: Remove ?

**Review by spaenleh — APPROVED** [review]
Looks good ! 🍫

File: src/items/LinkItem.stories.tsx (hunk 1 above)
Comment (pyphilia) [reply src/items/LinkItem.stories.tsx:7]: I've added it in the mui5 branch 👍

**Pull request merged**